bash_loading_animations.demo

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+source ./bash_loading_animations.sh
+
+demo_interval() { sleep 3 ; kill "${BLA_loading_animation_pid}" &> /dev/null ; echo "$1" ; }
```

Comment:
`sleep 7` was annoying me. Feel free to put it back if you so choose.

Reply:
I eventually tuned the sleep to be just long enough to see all the characters in each animation.

Reply:
Nice, I like it. I originally put 7 seconds because it was the shortest time needed to showcase more than half of the pong animation.
I expanded upon your code to more easily add new lines, and to make sure that the animations with a very short loop still get shown at least 3 seconds.